Index: package.json
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>{\r\n  \"name\": \"server\",\r\n  \"version\": \"1.0.0\",\r\n  \"description\": \"\",\r\n  \"main\": \"index.js\",\r\n  \"scripts\": {\r\n    \"dev\": \"nodemon\",\r\n    \"build\": \"tsc\",\r\n    \"serve\": \"node dist/server.js\",\r\n    \"test\": \"echo \\\"Error: no test specified\\\" && exit 1\"\r\n  },\r\n  \"keywords\": [],\r\n  \"author\": \"\",\r\n  \"license\": \"ISC\",\r\n  \"devDependencies\": {\r\n    \"@types/express\": \"^4.17.21\",\r\n    \"@types/node\": \"^20.14.2\",\r\n    \"dotenv\": \"^16.4.5\",\r\n    \"nodemon\": \"^3.1.0\",\r\n    \"ts-node\": \"^10.9.2\",\r\n    \"typescript\": \"^5.4.5\"\r\n  },\r\n  \"dependencies\": {\r\n    \"@fastify/swagger\": \"^8.14.0\",\r\n    \"@fastify/swagger-ui\": \"^3.1.0\",\r\n    \"@langchain/community\": \"^0.0.57\",\r\n    \"fastify\": \"^4.27.0\",\r\n    \"@fastify/cors\": \"^8.0.0\",\r\n    \"langchain\": \"^0.1.37\"\r\n  }\r\n}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/package.json b/package.json
--- a/package.json	(revision a8b45f78c41adedd306e8f384b2a56a33294b2b2)
+++ b/package.json	(date 1718750026783)
@@ -21,11 +21,11 @@
     "typescript": "^5.4.5"
   },
   "dependencies": {
+    "@fastify/cors": "^8.5.0",
     "@fastify/swagger": "^8.14.0",
     "@fastify/swagger-ui": "^3.1.0",
     "@langchain/community": "^0.0.57",
     "fastify": "^4.27.0",
-    "@fastify/cors": "^8.0.0",
     "langchain": "^0.1.37"
   }
-}
\ No newline at end of file
+}
diff --git a/src/routes/questions.ts b/src/services/routes/questions.ts
rename from src/routes/questions.ts
rename to src/services/routes/questions.ts
